core/templates/css/oppia.css

```diff
@@ -2119,7 +2119,7 @@ pre.oppia-pre-wrapped-text {
 .oppia-add-response-button,
 .oppia-add-fallback-button,
 .oppia-add-hint-button {
-  background-color: rgba(5,140,166,0.9);
+  background-color: rgba(5, 140, 166, 0.902);
```

Comment:
Why is this changed from 0.9 to 0.902?

Reply:
I don't why this changed 😅. I will undo it.

Reply:
Done!